### PR TITLE
Change paper-item focused bg color and selected appearance

### PR DESCRIPTION
**Pull request by blasten**
**Material design spec audit - few changes needed  https://github.com/PolymerElements/paper-menu/issues/11**
- Set background color for focused items and make them bold if selected.

cc @morethanreal @cdata @notwaldorf

### CODE REVIEW

File: paper-menu.html [hunk 1]
```diff
@@ -27,34 +27,39 @@
...
       position: relative;
+      outline: 0;
     }
 
-    :host > ::content > .iron-selected::after {
+    :host > ::content > *:focus:after {
       @apply(--layout-fit);
-
       background: currentColor;
       /* FIXME move to paper-styles for next widget */
```
Comment (cdata) [inline paper-menu.html:45]: Did you get an answer for what to do with FIXME?

Comment (blasten) [inline paper-menu.html:45]: @cdata yes.  I will add a `menu.css` to paper-styles. That was @morethanreal suggestion.

File: bower.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/PolymerElements/paper-menu",
   "ignore": [],
   "dependencies": {
-    "iron-menu-behavior": "PolymerElements/iron-menu-behavior#^0.9.0",
+    "iron-menu-behavior": "PolymerElements/iron-menu-behavior#^0.9.2",
```
Comment (notwaldorf) [inline bower.json:23]: You'll probably want to update this bower.json's version to 0.9.3, if you want to tag a release after merging this :)

**Comment by notwaldorf** [comment]
Hmmm, I'm probably overthinking this but: what if we wanted to allow the selected style to not be the same as the focused style? You've taken away all the `iron-selected` custom styles, so now they look like they're always the same. I can see how a user would want, for example: background + bold = selected, background + not bold = focused.

File: paper-menu.html (hunk 1 above)
Comment (morethanreal) [inline paper-menu.html:45]: How about

``` css
opacity: var(--paper-menu-focused-item-opacity-greyscale, --paper-focused-opacity-greyscale);`
```

and

``` css
opacity: var(--paper-menu-focused-item-opacity-colored, --paper-focused-opacity-colored);`
```

where `--paper-focused-opacity-greyscale` and `--paper-focused-opacity-colored` are defined by something like `common.html` in `paper-styles`?

Comment (morethanreal) [inline paper-menu.html:45]: But not a big deal if nobody else is currently depending on `opacity:0.12;` and `opacity:0.26;`

**Comment by morethanreal** [comment]
lgtm

File: paper-menu.html (hunk 1 above)
Comment (blasten) [inline paper-menu.html:45]: @morethanreal  I will be happy to do that in a new PR. thanks for merging!

**Pull request merged**